src/bp-blogs/bp-blogs-functions.php

```diff
@@ -1415,11 +1409,11 @@ function bp_blogs_add_blogmeta( $blog_id, $meta_key, $meta_value, $unique = fals
  * Remove all blog associations for a given user.
  *
  * @param int $user_id ID whose blog data should be removed.
- * @return bool Returns false on failure.
  */
 function bp_blogs_remove_data( $user_id ) {
-	if ( !is_multisite() )
-		return false;
+	if ( !is_multisite() ) {
```

Comment:
Let's add a space between ! & the function 😉

```suggestion
	if ( ! is_multisite() ) {
```

Reply:
I plan on tackling this in the whole project soon. 💪🏾